project/Dependencies.scala

```diff
@@ -5,7 +5,8 @@ object Dependencies {
     "Local Maven Repo"  at "file://" + Path.userHome.absolutePath + "/.m2/repository",
     "Typesafe Repo"     at "http://repo.typesafe.com/typesafe/releases/",
     "Sonatype releases" at "https://oss.sonatype.org/content/repositories/releases",
-    Resolver.bintrayRepo("mfglabs", "maven")
+    Resolver.bintrayRepo("mfglabs", "maven"),
+    "Elasticsearch releases" at "https://maven.elasticsearch.org/releases/"
```

Comment:
I read somewhere (can't find it now) that "virtual repos" (which I'm guessing `/releases/` is) can cause the `[error] Unable to find credentials for [Artifactory Realm @ ...`.  I poked around at https://maven.elasticsearch.org, and found there was another path to the same artifacts at `/public-releases/`.  If you change the URL, the error goes away.  Why...I dunno.

The fix: `https://maven.elasticsearch.org/public-releases/`